sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawContainer.java

```diff
@@ -15,7 +16,12 @@ private RawContainer(
             @JsonProperty("image-name") String imageName,
             @JsonProperty("rlimits") WriteOnceLinkedHashMap<String, RawRLimit> rlimits) {
         this.imageName = imageName;
-        this.rlimits = rlimits;
+
+        if (rlimits != null) {
```

Comment:
Those checks are for emptiness, not non-nullness, right?  All else equal, I think it's better to get rid of nulls early.